edx-platform/pearson-pathways-theme/lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -496,18 +496,26 @@ <h4 class="message-title">${_('Your current verification will expire soon.')}</h
         % endif
       % endif
 
-        % if course_requirements:
-        ## Multiple pre-requisite courses are not supported on frontend that's why we are pulling first element
-        <% prc_target = reverse('about_course', args=[six.text_type(course_requirements['courses'][0]['key'])]) %>
-        <div class="prerequisites">
-          <p class="tip">
-            ${Text(_("You must successfully complete {link_start}{prc_display}{link_end} before you begin this course.")).format(
-                link_start=HTML('<a href="{}">').format(prc_target),
-                link_end=HTML('</a>'),
-                prc_display=course_requirements['courses'][0]['display'],
-              )}
-          </p>
-        </div>
+       % if course_requirements:
+          % if student_not_enrollment_in_requirement:
+            ## Multiple pre-requisite courses are not supported on frontend that's why we are pulling first element
+            <div class="prerequisites">
+                You must successfully complete <a href="${ecommerce_payment_page}?sku=${student_not_enrollment_in_requirement}">${Text(_("{prc_display}")).format(
```

Comment:
Why don't you use the pattern used in line 511?. That is for marking strings as a translation string. @anfbermudezme